components/footer-simple.html

```diff
@@ -0,0 +1,103 @@
+<footer class="text-gray-700 body-font">
```

Comment:
Maybe just add an overall background to the whole footer instead?

```suggestion
<footer class="text-gray-700 bg-gray-400 body-font">
```